homeassistant/components/netgear/device_tracker.py

```diff
@@ -1,170 +1,134 @@
 """Support for Netgear routers."""
 import logging
-from pprint import pformat
-
-from pynetgear import Netgear
-import voluptuous as vol
-
-from homeassistant.components.device_tracker import (
-    DOMAIN,
-    PLATFORM_SCHEMA,
-    DeviceScanner,
-)
-from homeassistant.const import (
-    CONF_DEVICES,
-    CONF_EXCLUDE,
-    CONF_HOST,
-    CONF_PASSWORD,
-    CONF_PORT,
-    CONF_SSL,
-    CONF_USERNAME,
-)
-import homeassistant.helpers.config_validation as cv
+from typing import Dict
 
-_LOGGER = logging.getLogger(__name__)
-
-CONF_APS = "accesspoints"
-
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
Let's keep the platform schema until we remove all yaml support completely. Just log a warning if we find config yaml.

Reply:
We don't use `CONF_DEVICES`, `CONF_EXCLUDE` and `CONF_APS` anymore.

Should we keep it for posterity until we remove the config flow or I can remove it now (with `extra=vol.ALLOW_EXTRA`) ?

Reply:
Please keep it intact.

Reply:
:ok_hand: